docs/admin/templates/extending-templates/prebuilt-workspaces.md

```diff
@@ -173,23 +175,44 @@ resource "docker_container" "workspace" {
+
+### Current limitations
+
+The prebuilt workspaces feature has these current limitations:
+
+- **Organizations**
+
+  Prebuilt workspaces can only be used with the default organization.
+
+  [coder/internal#364](https://github.com/coder/internal/issues/364)
+
+- **Autoscaling**
+
+  Prebuilt workspaces remain running until claimed. There's no automated mechanism to reduce instances during off-hours.
+
+  [coder/internal#312](https://github.com/coder/internal/issues/312)
+
+### Monitoring and observability
+
+#### Available metrics
+
+Coder provides several metrics to monitor your prebuilt workspaces:
+
+- `coderd_prebuilt_workspaces_created_total` (counter): Total number of prebuilt workspaces created to meet the desired instance count.
+- `coderd_prebuilt_workspaces_failed_total` (counter): Total number of prebuilt workspaces that failed to build.
+- `coderd_prebuilt_workspaces_claimed_total` (counter): Total number of prebuilt workspaces claimed by users.
+- `coderd_prebuilt_workspaces_desired` (gauge): Target number of prebuilt workspaces that should be available.
+- `coderd_prebuilt_workspaces_running` (gauge): Current number of prebuilt workspaces in a `running` state.
+- `coderd_prebuilt_workspaces_eligible` (gauge): Current number of prebuilt workspaces eligible to be claimed.
 
-### Metrics
+#### Logs
 
-- `coderd_prebuilt_workspaces_created_total` (counter): Total number of prebuilt workspaces that have been created to
-  meet the desired instance count of each template preset
-- `coderd_prebuilt_workspaces_failed_total` (counter): Total number of prebuilt workspaces that failed to build
-- `coderd_prebuilt_workspaces_claimed_total` (counter): Total number of prebuilt workspaces which were claimed by users.
-  Claiming refers to creating a workspace with a preset selected for which eligible prebuilt workspaces are available
-  and one is reassigned to a user
-- `coderd_prebuilt_workspaces_desired` (gauge): Target number of prebuilt workspaces that should be available for each
-  template preset
-- `coderd_prebuilt_workspaces_running` (gauge): Current number of prebuilt workspaces that are in a running state. These
-  workspaces have started successfully but may not yet be claimable by users (see `coderd_prebuilt_workspaces_eligible`)
-- `coderd_prebuilt_workspaces_eligible` (gauge): Current number of prebuilt workspaces that are eligible to be claimed
-  by users. These are workspaces that have completed their build process with their agent reporting 'ready' status
+Search for `coderd.prebuilds:` in your logs to track the reconciliation loop's behavior.
 
-### Logs
+These logs provide information about:
 
-Search for `coderd.prebuilds:` to gain insight into the behavior of the reconciliation loop.
+1. Creation and deletion attempts for prebuilds.
+1. Backoff events after failed builds.
+1. Eligibility state changes.
```

Comment:
I don't think this is true anymore; where did you derive this from?

Reply:
this was part of the guess from [available metrics](https://github.com/coder/coder/pull/17580/files#r2067399290)

the metrics are available but don't show in the logs?
```suggestion
```